nifi-extension-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/db/TranslationStrategy.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.processors.standard.db;
+
+import org.apache.nifi.components.DescribedValue;
+
+/**
+ * Enumeration of supported Database column name Translation Strategies
+ */
+public enum TranslationStrategy implements DescribedValue {
+    REMOVE_UNDERSCORE("Remove Underscore",
+            "Underscores '_' will be removed from column names Ex: 'Pics_1_23' becomes 'PICS123'"),
+    REMOVE_SPACE("Remove Space",
+            "Spaces will be removed from column names Ex. 'User Name' becomes 'USERNAME'"),
+    REMOVE_ALL_SPECIAL_CHAR("Remove Regular Expression Characters",  "Remove Regular Expression Characters"),
```

Comment:
Do you think we should allow not normalizing the upper/lower case? Also since the list isn't long it might be worthwhile to add a "Remove Underscores and Spaces" option. Regular Expressions can get pretty gnarly and the more use cases we can anticipate the better.

Reply:
Hi @mattyb149,

I believe we should allow the option to normalize the upper/lower case. This can be very useful in scenarios where the source database allows case-sensitive column names, but the target database does not.

Example: Migrating from MSSQL to Postgres.

Regarding the removal of underscores and spaces, I suggest handling these separately, as there are cases where users may not want to remove underscores.

Example: If the source has column names `col_1_11 `and `col_11_1`, and we remove underscores, both columns would become col111, which could lead to conflicts.

Please let me know your thoughts.

Reply:
If they don't want to do both can't they pick one of the other options? I'm not suggesting removing any options but adding one.

Reply:
yes you are right that will help .  i will add "Remove Underscores and Spaces" option

Reply:
@mattyb149 I have new strategy `REMOVE_UNDERSCORE_AND_SPACE  `.
Regarding  lower case and upper case  I think we should have separate property . This option  will be visible only if normalization is rue.  if you are agree  then i will create new PR for that and work on it.
let me know your thought

Reply:
@mattyb149  let me know if there is any change required from my side